### PR TITLE
meal & grocery requests by ID #487 Fix

### LINKED ISSUE #487: Support searching meal & grocery requests by ID
Organizers would like to be able to search meal requests and grocery requests by ID in the search bar, Django supports this type of thing with `ModelAdmin#search_fields`

### PR DESCRIPTION
**Pull request by tibet5**
Fixes #487

### CODE REVIEW

File: website/recipients/admin.py [hunk 1]
```diff
@@ -131,9 +131,10 @@ class MealRequestAdmin(admin.ModelAdmin):
         'notify_deliverers_details',
     )
     search_fields = (
+        '=id',
         'name',
-        'email',
-        'phone_number',
+        '=email',
+        '=phone_number',
```
Comment (ryanwilsonperkin) [inline website/recipients/admin.py:137]:
```suggestion
        'email',
        'phone_number',
```

Let's keep these ones as generically searchable, often an organizer will want to search by just one part of the phonenumber/email.

File: website/recipients/admin.py [hunk 2]
```diff
@@ -331,9 +332,10 @@ class GroceryRequestAdmin(admin.ModelAdmin):
         GroceryRequestCommentInline,
     )
     search_fields = (
+        '=id',
         'name',
-        'email',
-        'phone_number'
+        '=email',
+        '=phone_number'
```
Comment (ryanwilsonperkin) [inline website/recipients/admin.py:338]:
```suggestion
        'email',
        'phone_number'
```

**Review by ryanwilsonperkin — CHANGES_REQUESTED** [review]
Thank you for the PR! I have one request about changing back email & phone number, but I like the approach taken with exact search on ID.

**Review by ryanwilsonperkin — APPROVED** [review]
LGTM

Thanks for your help! Now that this is approved you should be able to merge it (let me know if it prevents you) and it will automatically deploy.

**Comment by tibet5** [comment]
it says that some checks haven't completed yet. How do i complete? Sorry for this chat like comments this is my first contribute to an open source project

**Comment by ryanwilsonperkin** [comment]
No worries, you're doing great. This looks like a bug with GitHub itself: it should be showing me a button to "approve and run" the checks against your Pull Request, but there's no button. I've opened a bug report for them here: https://github.community/t/missing-approve-and-run-button/200572

In the meantime, I'll go ahead and merge this PR

**Pull request merged**